docker/kubernetespy/dynamic_version.sh

```diff
@@ -0,0 +1 @@
+pip freeze | grep demisto-sdk | grep -o -E '[0-9]{1,2}\.[0-9]{1,2}(\.[0-9]+)?'
```

Comment:
looks to me like this is failing the build. need to change demisto-sdk to kubernetes

Reply:
oh geez what an over sight on my end!